raiden/raiden_event_handler.py

```diff
@@ -59,6 +59,7 @@
 from raiden.transfer.views import get_channelstate_by_token_network_and_partner
 from raiden.utils.packing import pack_balance_proof_update, pack_withdraw
 from raiden.utils.typing import MYPY_ANNOTATION, Address, BlockSpecification, Nonce
+from raiden_contracts.tests.utils.constants import LOCKSROOT_OF_NO_LOCKS
```

Comment:
Doesn't have to be done with this PR, but if we use `LOCKSROOT_OF_NO_LOCKS` instead of `EMPTY_MERKLE_ROOT` now, it should be moved to `raiden_contracts.constants`. Always importing from `raiden_contracts.tests.utils` in production code is awkward.

Reply:
This is something that we block from merging rather than merge and leave to another PR IMO